barretenberg/cpp/src/barretenberg/commitment_schemes/gemini/gemini.hpp

```diff
@@ -99,20 +99,24 @@ template <typename Curve> class GeminiProver_ {
     using Claim = ProverOpeningClaim<Curve>;
 
   public:
-    static std::vector<Polynomial> compute_fold_polynomials(std::span<const Fr> multilinear_evaluations,
+    static std::vector<Polynomial> compute_fold_polynomials(const size_t log_N,
+                                                            std::span<const Fr> multilinear_challenge,
```

Comment:
We were passing the challenge in usages of this function but in the declaration the signature made it seem like we use the polynomial evaluations